idd/Energy+.idd.in

```diff
@@ -52147,6 +52147,11 @@ UnitarySystemPerformance:Multispeed,
        \maximum 10
        \note Used only for Multi speed coils
        \note Enter the number of the following sets of data for air flow rates.
+  A2 , \field Single Mode Operation
+       \type choice
```

Comment:
\note Controls coil operation during each HVAC timestep. 
\note Yes = operate at the highest speed possible without exceeding the current load.
\note No = allow operation at the average of two speeds to match the current load.
*Feel free to reword this*